src/deploy/NVA_build/logrotate_noobaa.conf

```
@@ -40,4 +40,7 @@
         dateformat %Y-%m-%d-%H-%M-%S
         sharedscripts
         olddir /var/log/noobaa_logs
-}
\ No newline at end of file
+        postrotate
+                kill -HUP $(pidof rsyslogd) >/dev/null 2>&1 || true
```

why not do something like `/bin/systemctl restart rsyslog` ? 
```suggestion
                /bin/systemctl restart rsyslog
```

---

While going through the man page of signal and rsyslogd, I understood that SIGHUP will not terminate the rsyslogd and will just re load its conf file. While in restart process there would be some time when rsyslogd will be in shutdown state. That plus the way we were doing it previously suggested to use HUP instead of restart.

While writing this, I just realized that I should also make this change for noobaa.log and client_noobaa.log files in this conf file. Will refresh it again.